SU-HKKU/cardinal_service_profile
# CSD-81 Mission statement paragraph type with basic pattern and styles

# READY FOR REVIEW

# Summary
- Mission statement paragraph type and stubbed out pattern.

# Need Review By (Date)
- 5/22

# Urgency
- medium

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-81`
1. `blt drupal:sync --site=cardinalservice`
1. edit or create a basic page
1. add the "mission statement" tool and populate the fields
1. verify it works. styling is done in another task.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
